DSharpPlus.CommandsNext/Attributes/RequireBotPermissionsAttribute.cs

```diff
@@ -51,6 +51,6 @@ public override async Task<bool> ExecuteCheckAsync(CommandContext ctx, bool help
 
         DiscordPermissions pbot = ctx.Channel.PermissionsFor(bot);
 
-        return pbot.HasAllPermissions(this.Permissions);
+        return pbot.HasAllPermissions([..this.Permissions]);

```

Comment:
Should we consider adding a `DiscordPermissions` overload instead of only a `ReadOnlySpan<DiscordPermissions>` overload?

Reply:
that is exactly what we have here, lunar

Reply:
This comment was made without the usage of intellisense and under the presumption that `this.Permissions` was `DiscordPermissions` and not `DiscordPermission[]`